assets/queries/openAPI/security_requirement_object_with_wrong_scopes/query.rego

```diff
@@ -0,0 +1,24 @@
+package Cx
+
+import data.generic.openapi as openapi_lib
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openapi_lib.check_openapi(doc) != "undefined"
+	[path, value] := walk(doc)
+	sec := value.security[idx][name]
```

Comment:
I think we should create a rule for operation object, since it [uses Security Requirement Object](https://swagger.io/specification/#operation-object) as well